Detected_Object detected_object
float32[]       words_histogram
